resources/examples/standalone/plot_energy_probability.py

```diff
@@ -7,7 +7,7 @@
 from matplotlib.colors import LogNorm
 
 import numpy as np
-
+from tqdm import tqdm
```

Comment:
Does proposal have a dependency on `tqdm`?

Reply:
I would not say that examples contribute to "dependencies", PROPOSAL does not depend on matplotlib or numpy either.

I would also remove these "try except ImportError" lines, `ImportError no module named matplotlib` will be pretty clear to every python user, and not really worse that `ImportError you need to install matplotlib`